### PR TITLE
[CON-5195] user is now notified why a order can't be delivered

### PR DESCRIPTION
**Pull request by SebastianFranze**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   },
   "require": {
     "firebase/php-jwt": "^3.0",
-    "shopware/connect-sdk": "dev-master#v2.0.12"
+    "shopware/connect-sdk": "dev-CON-5195"
```
Comment (SebastianFranze) [inline composer.json]: Don't merge until sdk is released

File: Snippets/frontend.sql [hunk 2]
```diff
@@ -66,5 +66,6 @@ ON DUPLICATE KEY UPDATE
   `namespace` = VALUES(`namespace`),
   `shopID` = VALUES(`shopID`),
   `name` = VALUES(`name`),
-  `localeID` = VALUES(`localeID`)
+  `localeID` = VALUES(`localeID`),
+  `value` = VALUES(`value`)
```
Comment (SebastianFranze) [inline Snippets/frontend.sql]: I don't know if this is a problem. Users could overwrite our snippets. With this new Update we would overwrite it back to our snippets every time the plugin is installed (updated)

File: Components/BasketHelper.php [hunk 3]
```diff
@@ -664,6 +668,24 @@ public function getConnectTemplateVariables(array $connectMessages)
         ];
     }
 
+    private function guessProductTitle(Message $message)
```
Comment (JanPietrzyk) [inline Components/BasketHelper.php]: mach ma `determineProductTitles`, wa?

File: Snippets/frontend.sql (hunk 2 above)
Comment (JanPietrzyk) [reply Snippets/frontend.sql]: weil der Rest der Datei ja ein ähnliches Problem hat, find ich das zwar Schade, aber nicht soooo schlimm. Hat es einen Grund warum Connect die Snippets nicht in den ini-Files hat?

**Pull request merged**